gateway/gateway.go

```diff
@@ -16,6 +16,7 @@ import (
 	"sync/atomic"
 	"time"
 
+	"github.com/rudderlabs/rudder-server/enterprise/pathfinder"
```

Comment:
remove this explicitly here and pass pathfinder (using application.Features().Pathfinder or something) from main.go ?

Reply:
This will go away. Added for testing purposes

Reply:
Infact this failed our oss build :)